### ISSUE TITLE
unexpected warning in console for igx-date-picker 

### ISSUE THREAD

**Issue by loclv** [IgniteUI/igniteui-angular#9089]
## Unexpected warning in console for igx-date-picker  

![Screenshot 2021-02-23 172236](https://user-images.githubusercontent.com/6302717/109387036-646aa080-7931-11eb-96d4-f76567e42c66.png)

Unexpected warning:

`IgxDatePickerComponent.label: Use igxLabel inside the date picker to change the label:`

```html
    <igx-date-picker>
        <label igxLabel>Custom label</label>
    </igx-date-picker>
```

 * igniteui-angular version: same with [igniteui-angular-samples](https://github.com/IgniteUI/igniteui-angular-samples)
 * browser: lastest chrome

## Steps to reproduce  

1. Start development server of  [igniteui-angular-samples](https://github.com/IgniteUI/igniteui-angular-samples) with `yarn start`.
2. Goto`http://localhost:4200/samples/scheduling/datepicker-sample-1`.
3. Open browser console.

## Result 

You can see that warning.

## Expected result

No warning, with basic using.

**Comment by wnvko** [comment]
Related to #8917 
This should be already fixed in 11.0.14 and 10.2.14.

**Comment by loclv** [comment]
@wnvko thank you!
